src/capital-project/capital-project.repository.ts

```diff
@@ -50,6 +56,32 @@ export class CapitalProjectRepository {
           category: sql<CapitalProjectCategory>`${capitalProject.category}`,
         })
         .from(capitalProject)
+        .leftJoin(
+          cityCouncilDistrict,
+          sql`
+            ST_Intersects(${cityCouncilDistrict.liFt}, ${capitalProject.liFtMPoly})
+            OR ST_Intersects(${cityCouncilDistrict.liFt}, ${capitalProject.liFtMPnt})`,
+        )
+        .leftJoin(
+          communityDistrict,
+          sql`
+          ST_Intersects(${communityDistrict.liFt}, ${capitalProject.liFtMPoly})
+          OR ST_Intersects(${communityDistrict.liFt}, ${capitalProject.liFtMPnt})`,
+        )
+        .where(
+          and(
+            ccd ? eq(cityCouncilDistrict.id, ccd) : undefined,
```

Comment:
More explicit check for missing district values: (repeat for community district + borough Id)

```suggestion
            ccd !== null ? eq(cityCouncilDistrict.id, ccd) : undefined,
```